### PR TITLE
Workflow Automated Test - LRQA-68602 | master

### PR DESCRIPTION
**Pull request by PauloVAS**

### CODE REVIEW

**Comment by PauloVAS** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowCalendar.testcase [hunk 1]
```diff
@@ -170,12 +170,105 @@ definition {
...
 
 	@description = "Verify that a calendar event on draft can be approved"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CalendarWithDraftStatusCanBecomeApproved {
+		Navigator.gotoPage(pageName = "Calendar Page");
+
+		CalendarNavigator.gotoAddEvent();
+
+		eventTitle = "Test Event";
```
Comment (brittneyq) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowCalendar.testcase:179]: Hi @PauloVAS, It looks like this macro is incomplete, would you mind fixing this?

**Comment by PauloVAS** [comment]
ci:test:sf

**Comment by PauloVAS** [comment]
@brittneyq, fixed!

**Comment by brittneyq** [comment]
ci:forward

**Pull request closed without merging**